docker-compose.yml

```diff
@@ -51,12 +51,16 @@ services:
       CROWN_API_PASSWORD: ${CROWN_API_PASSWORD:? CROWN_API_PASSWORD not set}
       CROWN_WITHDRAW_FEE_INT: ${CROWN_WITHDRAW_FEE_INT:? CROWN_WITHDRAW_FEE_INT not set}
       CROWN_WITHDRAW_NAME: ${CROWN_WITHDRAW_NAME:? CROWN_WITHDRAW_NAME not set}
+      MAIL_SERVER: ${MAIL_SERVER:? MAIL_SERVER not set}
+      MAIL_PORT: ${MAIL_PORT:? MAIL_PORT not set}
+      USE_SENDGRID: ${USE_SENDGRID}
```

Comment:
get rid of this, we can infer it from the sendgrid api key value

also change the sendgrid config to be optional:
`SENDGRID_API_KEY: ${SENDGRID_API_KEY:?SENDGRID_API_KEY unset}`
to
`SENDGRID_API_KEY: ${SENDGRID_API_KEY}`